LayoutTests/accessibility/mac/line-text-marker-range-for-text-marker.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../resources/accessibility-helper.js"></script>
+<script src="../../resources/js-test.js"></script>
+</head>
+<body id="body">
+
+<div id="textcontrol" contenteditable="true">
+  First line of text
+  <span contenteditable="false"><br></span>
+  Second line of text
+</div>
+
+<script>
+let output = "This tests that when there are &lt;br&gt; elements with contenteditable=false in a contenteditable text area, we can get the correct range for a line.\n\n";
+
+if (window.accessibilityController) {
+  let textControl = accessibilityController.accessibleElementById("textcontrol");
+  let fullRange = textControl.textMarkerRangeForElement(textControl.childAtIndex(0));
+  let marker = textControl.startTextMarkerForTextMarkerRange(fullRange);
+  let range = textControl.lineTextMarkerRangeForTextMarker(marker);
+  window.rangeStr = (textControl.stringForTextMarkerRange(range) + "").trim();
+  output += expect("rangeStr", '"First line of text"');
```

Comment:
I think we tend towards using `var rangeStr` for things that need to be used in `expect` rather than assigning the variable to `window`. I think it's a bit more resilient in case `window` eventually gains a property with that name (though obviously probably not going to happen for `rangeStr`)

Reply:
Though maybe a `var` would conflict in that case too 🤔 I'm fine leaving this as-is or using `var`